### PR TITLE
Updated requirements

### PR DESCRIPTION
**Pull request by hawkinsw**
## Description

Bring `GettingStarted.md` into sync with *actual* requirements (i.e., `clang` is no longer required.) Just cleaning up a discrepancy that I caused! Sorry!

## Documentation

In the PR itself.

### CODE REVIEW

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -37,14 +37,17 @@ install it directly from [here](https://www.microsoft.com/en-us/download/details
...
     >Note: as multiple versions of WDKs cannot coexist side-by-side, you may be asked to uninstall previous versions.
-1. [Clang for Windows 64-bit](https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.1/LLVM-11.0.1-win64.exe) (version **11.0.1**).
- Note: clang versions 12 and higher are NOT yet supported, as they perform program optimizations that are incompatible with the PREVAIL verifier.
 1. [NuGet Windows x86 Commandline](https://www.nuget.org/downloads) (version **6.3.1 or higher**), which can be installed to a location
  such as "C:\Program Files (x86)\NuGet\".
 
 You should add the paths to `git.exe`, `cmake.exe` and `nuget.exe` to the Windows PATH environment variable after the software packages
  above have been installed.
 
+The following are optional components that you can install for additional functionality:
```
Comment (dthaler) [inline docs/GettingStarted.md:46]: PR #2673 only affected the cmake build.  What about the Visual Studio build?
I didn't see any changes to tests\sample\sample.vcxproj

Comment (hawkinsw) [reply docs/GettingStarted.md:46]: Well played! I did not realize that both a CMake and a "Solution" build existed together. I will take a look! Sorry!!

**Comment by hawkinsw** [comment]
> @hawkinsw can you please address the comments or let us know if you cannot. Thanks.

So sorry! A few meetings ago we discussed dropping support for the CMake build. I figured that we had done that and, therefore, this PR would be moot. If we are continuing to support CMake then I will absolutely update the PR. 

Sorry again!

cc @dahavey

**Comment by dthaler** [comment]
Cmake is still used to build bpf programs, which is why it's required as a prerequisite.   Closing this PR per Will's comment, but please re-open if it is still relevant.

**Pull request closed without merging**